src/silo_api/api.cpp

```diff
@@ -124,9 +130,20 @@ class SiloServer : public Poco::Util::ServerApplication {
       server.stop();
    };
 
-   // NOLINTNEXTLINE(readability-convert-member-functions-to-static)
    void handleProcessData() {
-      std::cout << "handleProcessData is not implemented\n" << std::flush;
+      SPDLOG_INFO("Starting SILO preprocessing");
+      const std::string preprocessing_config_path = preprocessingConfigPath(config());
+      const std::string database_config_path = databaseConfigPath(config());
+
+      auto preprocessing_config =
+         silo::preprocessing::PreprocessingConfigReader().readConfig(preprocessing_config_path);
+      auto database_config =
+         silo::config::ConfigRepository().getValidatedConfig(database_config_path);
+
+      auto database_preprocessing =
```

Comment:
```suggestion
      auto preprocessed_database =
```
I still find that name confusing. It's a `Database`, so my brain expects that the variable ends with `database`.